babel.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    [
+      "@babel/preset-react",
+      {
+        "runtime": "automatic"
+      }
+    ],
+    "@babel/preset-typescript"
+  ],
+  "plugins": [
+    "@babel/plugin-transform-runtime",
+    "babel-plugin-transform-vite-meta-env"
```

Comment:
Ah nice! Ended up using it?

Reply:
Actually that might be leftover as I didn't need to use `import.meta` for anything. Let me remove it and see if everything is working properly.